src/components/App/App.js

```diff
@@ -51,13 +56,22 @@ const App = () => {
           return (
           <>
               <Header userName={userName} />
-              <FriendPage id={id} />
+              <FriendPage userId={userId} id={id} />
           </>
           )
         }} />
       </Switch>
       </main>
   )
+  } else if (!error && !userName) {
+    return (
+      <h2>Your gift info is on the way...</h2>
```

Comment:
Unless it was bought on Amazon and was a fake vendor :stuck_out_tongue:

Reply:
I like the loading message you added!